bpfman-operator/config/bpfman-operator-deployment/kustomization.yaml

```diff
@@ -5,4 +5,4 @@ kind: Kustomization
 images:
 - name: quay.io/bpfman/bpfman-operator
   newName: quay.io/bpfman/bpfman-operator
-  newTag: latest
+  newTag: v0.4.0
```

Comment:
Why are we doing this? Will this need to be backed out once the release is tagged?

Reply:
This was an accident good catch!